tensorbay/opendataset/UrbanObjectDetection/__init__.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
need to add `UrbanObjectDetection` in `docs/source/reference/api/opendataset.rst`